src/ucx_van.h

```diff
@@ -895,6 +892,8 @@ class UCXVan : public Van {
         CHECK(msg.meta.addr != 0);
         rx_pool_->CacheLocalAddress(msg.meta.key, (char*)msg.meta.addr);
       }
+    } else {
+      msg.meta.val_len = 0;
```

Comment:
looks like msg.meta.val_len would be undefined for push response and pull request in this case.
This could lead to wrong handling in `PostRecvData`